src/components/contributors/listContainer/index.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-console */
+import React, { Component } from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  FlatList
+} from 'react-native';
+
+import ContributorListItem from './listItem';
+
+const styles = StyleSheet.create({
+  flex: {
+    display: 'flex',
+    flex: 1,
+  },
+});
+
+// const sortData = (dataArray) => {
+//   // sorts the array by join date
+//   dataArray.sort((firstItem, secondItem) => (firstItem.joined < secondItem.joined ? -1 : 1))
+//     // then sorts array by active status
+//     .sort((firstItem, secondItem) => (firstItem.active > secondItem.active ? -1 : 1));
+//   return dataArray;
+// };
+
+export default class ListContainer extends Component {
+  constructor() {
+    super();
+    this.state = {
+      contributorsData: [],
+      dataLoaded: false,
+      isSelected: true
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://api-dev.coding.garden/contributors')
```

Comment:
We decided to use async/await for all promises on the backend and enforced it with an eslint plugin. I think we should follow that on frontend as well.